galaxy_ng/app/access_control/statements/roles.py

```diff
@@ -1,67 +1,153 @@
-VIEWSETS = {
-    "CollectionViewSet": {
-        "LOCKED_ROLES": {
-            "galaxy.collection_admin": [
-                "galaxy.change_namespace",
-                "galaxy.delete_namespace",
-                "galaxy.view_namespace",
-                "galaxy.upload_to_namespace",
-                "ansible.delete_collection",
-            ],
-        }
+LOCKED_ROLES = {
+    "galaxy.content_admin": {
```

Comment:
@newswangerd did you intent to add a `signature_admin` with `can_sign_collections` perm that also exists in `content_admin`?